src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateIndexCommand.java

```diff
@@ -28,6 +28,10 @@ public record CreateIndexCommand(
             description = "Definition for created index for a column.",
             type = SchemaType.OBJECT)
         RegularIndexDefinitionDesc definition,
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+        @Nullable
+        @Schema(description = "Type for created index for a column.", type = SchemaType.STRING)
```

Comment:
I know this follows pattern of the `definition` but I think it'd be better written as something like

    "Type of the index to create."

and then also for "definition":

    "Definition of the index to create."